crates/testenv/Cargo.toml

```diff
@@ -18,6 +18,7 @@ workspace = true
 [dependencies]
 bdk_chain = { path = "../chain", version = "0.19.0", default-features = false }
 electrsd = { version = "0.28.0", features = [ "bitcoind_25_0", "esplora_a33e97e1", "legacy" ] }
+bitcoin = { version = "0.32.0", default-features = false }
```

Comment:
Hello! Since `bdk_chain` already re-exports `bitcoin`, we don't need to add `bitcoin` as a dependency here.

Reply:
Fixed. Thank you